go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/adjust/rmq
```

Comment:
Can we do this with naming the module `github.com/adjust/rmq/v2`. The update will be a breaking change for projects who haven't yet switched to Go modules, because of `github.com/go-redis/redis/v7`

Reply:
Done!